src/scribe.js

```diff
@@ -71,8 +72,10 @@ define([
     var TransactionManager = buildTransactionManager(this);
     this.transactionManager = new TransactionManager();
 
-    var UndoManager = buildUndoManager(this);
-    this.undoManager = new UndoManager();
+    if (this.options.shouldUndo) {
+      var UndoManager = buildUndoManager(this);
+      this.undoManager = new UndoManager();
```

Comment:
I feel undoManager should be `undefined` and then take on a value if the options say it should do. Does that make sense?